src/ci/docker/run.sh

```diff
@@ -45,20 +45,20 @@ else
     args="$args --env SCCACHE_DIR=/sccache --volume $HOME/.cache/sccache:/sccache"
 fi
 
-exec docker \
-  run \
```

Comment:
Blast my attempt to improve our ci by 100000% percent has been foiled!

(definition not intentional to leave this change in)